server/assessment/service/exportService.js

```diff
@@ -20,62 +18,91 @@ const EXPORT_TYPE = {
   CSV: 'csv',
 }
 
-const exportData = async (user, exportType = EXPORT_TYPE.JSON) => {
-  AccessControl.checkAdminAccess(user)
-
+// if simple flag is true, only return fra years, intervals and annualoutput without subfolders
+const exportData = async (exportType = EXPORT_TYPE.JSON, simple) => {
```

Comment:
call the second param includeSubFolders?

Reply:
you mean excludeSubFolders?